tgui/packages/tgui-dev-server/package.json

```diff
@@ -1,13 +1,16 @@
 {
   "private": true,
   "name": "tgui-dev-server",
-  "version": "5.0.0",
+  "version": "5.0.3",
   "type": "module",
   "dependencies": {
     "axios": "^1.8.2",
-    "glob": "^7.1.7",
+    "glob": "^10.3.10",
```

Comment:
This breaks shit don't update it

Reply:
[70ee033](https://github.com/BeeStation/BeeStation-Hornet/pull/12784/commits/70ee03361442a6e4542bf5e7b1bd562a561266a7)